### DISCUSSION TITLE
No query results for model [App\Media].

### DISCUSSION THREAD

**Discussion by vanthao03596** [spatie/laravel-medialibrary#1815] [Q&A]
I'm using laravel-medialibrary v8 and laravel 7 and got error "No query results for model [App\Media]." when using queue with redis connection. Please help me to fix.
Here is my Conversions
![1](https://user-images.githubusercontent.com/34786441/78361173-2b6f6200-75e2-11ea-840e-e2bfa5ab0198.png)
Here is Supervisor 
![2](https://user-images.githubusercontent.com/34786441/78361334-71c4c100-75e2-11ea-940f-e58ec35f3a8f.png)
failed_jobs table
![1](https://user-images.githubusercontent.com/34786441/78361380-84d79100-75e2-11ea-875a-8c0a43e31eae.png)
P

**Comment by getupkid** [comment]
Have you added `use Spatie\MediaLibrary\Models\Media` to your model?

**Comment by vanthao03596** [comment]
I'm using custom model which extend Spatie\MediaLibrary\MediaCollections\Models\Media.
After change queue driver to database, everything work fine and no more fail job. I dont know why it has issue with redis
